### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by d-beschastnyi**

### CODE REVIEW

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]
image
fix tests

File: app/main.py [hunk 1]
```diff
@@ -16,4 +16,24 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [],
+    "immutable": []
+}
+
+for i in (
```
Comment (vasylhnatiuk) [inline app/main.py:24]:
```suggestion
for data in (
```

Comment (vasylhnatiuk) [reply app/main.py:24]: data is a bad name

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

**Review by vsmutok — APPROVED** [review]
Looks Good 👍

**Pull request closed without merging**